js/import-export.js

```diff
@@ -48,7 +48,7 @@ function _getFlexibleEntries() {
         const [year, month, day] = key.split('-');
         //The main database uses a JS-based month index (0-11)
         //So we need to adjust it to human month index (1-12)
-        const date = year + '-' + (parseInt(month) + 1) + '-' + day;
+        const date = generateKey(year, (parseInt(month) - 1), day);
```

Comment:
Looks like a copy and paste error here. It would be `+`:

```suggestion
        const date = generateKey(year, (parseInt(month) + 1), day);
```